postgres-driver/application.go

```diff
@@ -574,3 +577,22 @@ func (d *PostgresDriver) UpdateApplication(id string, fieldsToUpdate *repository
 
 	return tx.Commit()
 }
+
+// RemoveApplication updates fields available in options in db
+func (d *PostgresDriver) RemoveApplication(id string) error {
+	if id == "" {
+		return ErrMissingID
+	}
+
+	tx, err := d.Beginx()
+	if err != nil {
+		return err
+	}
+
+	_, err = tx.Exec(removeApplication, newSQLNullString("AWAITING_GRACE_PERIOD"), time.Now(), id)
```

Comment:
It's better to use `repository.AwaitingGracePeriod` to use the `AppStatus` type

Reply:
OK will set to use the enum